qiskit_machine_learning/algorithms/trainable_model.py

```diff
@@ -141,9 +142,42 @@ def initial_point(self, initial_point: np.ndarray) -> None:
         """Sets the initial point"""
         self._initial_point = initial_point
 
+    @property
+    def weights(self) -> np.ndarray:
+        """
+        Returns:
+            an array of trained weights. This property is for convenience only, it does the same
+            as `model.fit_result.x`.
+
+        Raises:
+            QiskitMachineLearningError: If the model has not been fit.
+        """
+        self._is_fit()
+        return np.asarray(self._fit_result.x)
+
+    @property
+    def fit_result(self) -> OptimizerResult:
+        """
+        Returns:
+            A resulting object from the optimization procedure. Please refer to the documentation of
+            the `OptimizerResult
```

Comment:
Similar issue to weights in regards of sphinx output

Reply:
Updated